package.json

```diff
@@ -120,7 +121,8 @@
     "@babel/preset-typescript": "^7.15.0",
     "@faker-js/faker": "^7.3.0",
     "@redhat-cloud-services/eslint-config-redhat-cloud-services": "^1.2.1",
-    "@redhat-cloud-services/frontend-components-config": "^4.6.9",
+    "@redhat-cloud-services/frontend-components-config": "^4.6.22",
+    "@redhat-cloud-services/frontend-components-testing": "^0.1.1",
```

Comment:
What is this package for? I feel like in an update story we shouldn't really add new packages